README.md

```diff
@@ -30,14 +30,14 @@ You can find the latest news about the annotation tool on [this page](documentat
 * [Claudio Beffa](beffa.ch)
 
 ### Current Development of this branch
-* [virtUOS, Universit�t Osnabr�ck](http://www.virtuos.uni-osnabrueck.de)
+* [virtUOS, Universität Osnabrück](http://www.virtuos.uni-osnabrueck.de)
```

Comment:
I'm confused. What exactly is the difference here? Is this some kind of encoding snafu? :thinking:

Reply:
I used the GitHub online editor and apparently it changed the encoding from ISO-8859-1 to UTF-8. Which one do you prefer?

Reply:
Oh god, so what you're saying is basically that I need to check every file's encoding now? x)

UTF-8 is what we want, I'd say.